src/views/Layout.jsx

```diff
@@ -1,4 +1,5 @@
 import { Outlet } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
 
```

Comment:
You can combine these into a single import

Reply:
Thanks for pointing that out!